src/extensions/surveys/surveys-extension-utils.tsx

```diff
@@ -633,6 +634,7 @@ export const dismissedSurveyEvent = (survey: Survey, posthog?: PostHog, readOnly
         $survey_questions: survey.questions.map((question) => ({
             id: question.id,
             question: question.question,
+            response: question.id ? inProgressSurvey?.responses[getSurveyResponseKey(question.id)] : null,
```

Comment:
do we need this in survey dismissed?

Reply:
since we show the same pattern for both right now, i'd like to keep it the same way. this is for the case of getting the responses in case the user dismisses the survey after answering a couple of questions